libraries/SITL/SIM_last_letter.cpp

```diff
@@ -146,6 +129,10 @@ void last_letter::update(const struct sitl_input &input)
     send_servos(input);
     recv_fdm(input);
     sync_frame_time();
+
+    update_position();
```

Comment:
Originally `update_position` wasn't called. I added this after the Gazebo SITL example, but I'm not sure what it does. I suspect that it updates the position variables on the SITL backend where the APM HAL sensors read from.